.github/workflows/flake8.yml

```diff
@@ -0,0 +1,226 @@
+name: Python code quality check
+
+on:
+- push
+- pull_request
+
+jobs:
+
+  flake8-grass7-db-scripts:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8==3.8.0
+    - name: Run Flake8
+      run: |
+        cd grass7/db
+        flake8 --config=../.flake8 --count --statistics --show-source --jobs=$(nproc) .
```

Comment:
You can reduce the duplicity further with matrix as in [gcc.yml](https://github.com/OSGeo/grass/blob/master/.github/workflows/gcc.yml#L12) in core:

```
flake8-job:
  strategy:
    matrix:
      directory:
      - db
      - display
      ...

     cd grass7/${{ matrix.directory }}
```

Reply:
Great suggestion, @wenzeslaus - seems to run much faster now after 6f28a0fbb7aabd245de51aac4b5f86330bd8ba6d